src/main/java/com/skynest/tests/tests/RegistrationTest.java

```diff
@@ -0,0 +1,33 @@
+package com.skynest.tests.tests;
+
+import com.skynest.tests.files.EndpointConstants;
+import com.skynest.tests.files.Payload;
+import com.skynest.tests.files.ReusableMethods;
+import io.restassured.path.json.JsonPath;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import static io.restassured.RestAssured.*;
+
+public class RegistrationTest extends BaseTest{
+    @Test(dataProvider = "UserData")
+    void registering_new_valid_user_should_return_specified_response(String email, String pass, String name, String surname, String phoneNo, String address){
+        String response = given().log().all().header("Content-Type","application/json")
+                .body(Payload.userRegistrationData(email,pass,name,surname,phoneNo,address))
+                .when()
+                .post(EndpointConstants.registerUrl)
+                .then().log().all().assertThat().statusCode(200)
+                .extract().response().asString();
+
+        JsonPath js= ReusableMethods.rawToJson(response);
+        String id = js.getString("id");
+        System.out.println("The unique id of the registered user is: "+id);
```

Comment:
println not needed here

assert library would help here with printing difference between expectation/actual (if any)

Reply:
Ok, println will be removed. 
Assertions for all arguments (keys) from the response body will be added.